### ISSUE TITLE
missing the tense and voice in Greek participle verbs (and Latin verbs too)

### ISSUE THREAD

**Issue by monzug** [alpheios-project/inflection-tables#290]
lookup  κατασκευάσας     - see screenshot below

![verb-participle](https://user-images.githubusercontent.com/41396793/68480294-75564680-0202-11ea-98b5-d205a54ac0f5.JPG)

@balmas: I think because it has a case, which is overriding the voice and tense information.

Comparing with the latin verbs (e.g. reputantis), I realized that tense and voice are also missing from the latin morphology pop-up but we do show the conjugation which we don't have in Greek verbs and in Morpheus. to add more info, should we consider the stemtype?

**Comment by balmas** [comment]
@monzug Yes, we currently don't show tense and voice if there is a case. 
Are you suggesting we show it in both the popup and the table, or just the popup?

**Comment by monzug** [comment]
we had a discussion at that time. would say just pop-up? maybe vanessa could give her opinion. we can ask at the call. what do you think?

**Comment by vgorman1** [comment]
I am not entirely sure what the question is.
Certainly all participles, as verbal adjectives, have to be identified in the morphology by number, gender, and case (like an adjective) but also by tense and voice (like a verb).  That said, they decline like regular adjectives, depending on the tense and voice: some are 3rd declension adjectives and some are 3-ending 1st and 2nd declension adjectives.
So, if the morphological tool is doing its job identifying the form completely, then the inflection table should just be the standard adjective tables.

Vanessa B. Gorman
Professor of Ancient History
Department of History
619 Oldfather Hall
University of Nebraska-Lincoln
Lincoln, NE 68588-0327
https://vgorman1.github.io/

**Comment by vgorman1** [comment]
BTW, look at Smyth 394-395 on the subject of χράω and, the one I forgot, ζάω. :
394
Ten verbs in -αω show η where we expect ᾱ. These are διψῶ thirst, ζῶ live, πεινῶ hunger, κνῶ scrape, νῶ spin (rare), σμῶ wash, χοῶ give oracles, χοῶ am eager for (rare), χρῶμαι use, and ψῶ rub. See 641<https://grammars.alpheios.net/smyth/xhtml/body.1_div1.2_div2.10.html#s641>,
395
ζῶ live and χρῶμαι use are inflected as follows in the present indicative, subjunctive and imperative and in the imperfect.
Indic. and Subj.
Imperative
Imperfect
S. 1. ζῶ
χρῶμαι
ἔζων
ἐχρώμην
2. ζῇς
χρῇ
ζῆ
χρῶ
ἔζης
ἐχρῶ
3. ζῇ
χρῆται
ζήτω
χρήσθω
ἔζη
ἐχρῆτο
D. 2. ζῆτον
χρῆσθον
ζῆτον
χρῆσθον
ἐζῆτον
ἐχρῆσθον
3. ζῆτον
χρῆσθον
ζήτων
χρήσθων
ἐζήτην
ἐχρήσθην
P. 1. ζῶμεν
χρῶμεθα
ἐζῶμεν
ἐχρώμεθα
2. ζῆτε
χρῆσθε
ζῆτε
χρῆσθε
ἐζῆτε
ἐχρῆσθε
3. ζῶσι
χρῶνται
ζώντων
χρήσθων
ἔζων
ἐχρῶντο
Infinitive:
ζῆν, χρῆσθαι
Participle:
ζῶν, χρώμενος


Vanessa B. Gorman
Professor of Ancient History
Department of History
619 Oldfather Hall
University of Nebraska-Lincoln
Lincoln, NE 68588-0327
https://vgorman1.github.io/

**Comment by monzug** [comment]
vanessa, I have the smith next to me and consulting it for the to be verb.
will also look at the section above
